ledger-core/virtual/integration/vstatereport_test.go

```diff
@@ -29,79 +27,6 @@ import (
 	"github.com/insolar/assured-ledger/ledger-core/virtual/testutils"
 )
 
-// 1. Send CallRequest
-// 2. Change pulse in mocked executor
-// 4. Since we changed pulse during execution, we expect that VStateReport will be sent
-// 5. Check that in VStateReport new object state is stored
-func TestVirtual_SendVStateReport_IfPulseChanged(t *testing.T) {
-	t.Log("C4934")
```

Comment:
There are a lot of checks for VStateReport in other tests, for example: `TestVirtual_Method_CheckPendingsCount`, caseID = "C5104"